services/gcp/cloudsql/users.yml

```
@@ -0,0 +1,46 @@
+name: Google Cloud SQL
+description: >-
+  Cloud SQL users are used to authenticate into Cloud SQL databases.
+scope: CRITICAL
+notes: >-
+  Cloud SQL is used to store and serve sensitive and application-critical data. Breach of a cloud SQL database can lead to exfiltration of highly sensitive data, or interruption of mission-critical applications.
+privileges:
+  create:
+    risks:
+      [
+        destruction:data,
+        destruction:logs,
+        exfiltration:data,
+        exfiltration:logs,
+        impact:manipulation,
+      ]
```

Use YAML array syntax for multiline:
```suggestion
      - destruction:data
      - destruction:logs
      - exfiltration:data
      - exfiltration:logs
      - impact:manipulation
```